module-apps/application-calculator/tests/CalculatorInput_tests.cpp

```diff
@@ -478,6 +459,26 @@ SCENARIO("Input Processor tests")
                     REQUIRE(inputField.getText() == "56");
                 }
             }
+
+            AND_WHEN("We invoke any operation")
+            {
+                passShortKeyPress(KeyCodes::JoystickUp);
+
+                THEN("We should see")
+                {
+                    REQUIRE(inputField.getText() == "9.9998e9+");
+                }
+
+                AND_WHEN("We proceed to type more input")
+                {
+                    passShortKeyPresses({KeyCodes::NumericKey1, KeyCodes::NumericKey5, KeyCodes::NumericKey6});
+
+                    THEN("We should see")
```

Comment:
:clap: 
These are test names (and their sequence) that tell what they're doing, congrats!